scripts/ci/build_and_push_agent_image.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+
+# shellcheck disable=SC1091
+. venv/bin/activate
+echo "${quay_password:?}" | docker login "-u=${quay_user_name:?}" quay.io --password-stdin
+
+
+# Providing the quay.expires-after configures quay to delete this image after the provided amount of time
+docker build . --label "quay.expires-after=${expire_after:-never}" -f agent/Dockerfile -t "${image:?}" --build-arg tools_version="100.2.0" --build-arg agent_version="10.27.0.6772-1"
```

Comment:
these hard coded versions will be removed along with this script once in the next PR.